### DISCUSSION TITLE
Pushover integration

### DISCUSSION THREAD

**Discussion by blkbrd12** [containrrr/watchtower#1280] [Feature Requests] [answered]
I'm trying to integrate Watchtower with Pushover but the link you have to the documentation is broken.

**Comment by piksel** [comment, marked as answer]
What link? This is the documentation for the shoutrrr pushover service:
https://containrrr.dev/shoutrrr/v0.5/services/pushover/

This is for using shoutrrr notifications with watchtower:
https://containrrr.dev/watchtower/notifications/#containrrrshoutrrr

**Comment by blkbrd12** [comment]
Got it, thanks! I was looking at an old post in which somebody else asked the same question and the link was broken.

Sent from my iPhone

On May 1, 2022, at 3:04 AM, nils måsén ***@***.***> wrote:

﻿

What link? This is the documentation for the shoutrrr pushover service:
https://containrrr.dev/shoutrrr/v0.5/services/pushover/<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcontainrrr.dev%2Fshoutrrr%2Fv0.5%2Fservices%2Fpushover%2F&data=05%7C01%7C%7Cb3f054a1535547cf66b008da2b40e6bc%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637869854993701925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OEqo24%2F4ZqcxRUEONzBmNh1W9AWGCUpPhtIiDdx9srY%3D&reserved=0>

This is for using shoutrrr notifications with watchtower:
https://containrrr.dev/watchtower/notifications/#containrrrshoutrrr<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcontainrrr.dev%2Fwatchtower%2Fnotifications%2F%23containrrrshoutrrr&data=05%7C01%7C%7Cb3f054a1535547cf66b008da2b40e6bc%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637869854993701925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7vS2PuVG0Q1zeE39P3wXeuSsNfd539G0ch8qndx59UQ%3D&reserved=0>
